cmd/ls/ls.go

```diff
@@ -43,12 +45,30 @@ func formatFileName(name string) string {
 	return name
 }
 
+func lookupIds(uid, gid string) (string, string) {
+	u, err := user.LookupId(uid)
+	if err != nil {
+		fatal(err)
```

Comment:
full stack for who? the user that knows nothing about Go nor the project design? =P

I think LookupIds must return an error and this propagates to main deferring all functions along with it...